app-text/wgetpaste/metadata.xml

```diff
@@ -2,7 +2,15 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<maintainer type="person">
+		<email>nvinson234@gmail.com</email>
+		<name>Nicholas Vinson</name>
+	</maintainer>
+	<maintainer type="person">
 		<email>maksbotan@gentoo.org</email>
 		<name>Maxim Koltsov</name>
 	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
```

Comment:
Same here, it is nice that you want to help. However the package has an active Gentoo developer. You cannot add the proxy-maintenance project by yourself.